composer.json

```diff
@@ -47,17 +48,46 @@
         }
     },
     "scripts": {
-         "phpstan": [
-             "Tests/Application/bin/console cache:warmup --env dev",
-             "vendor/bin/phpstan analyse"
-         ],
-         "bootstrap-test-env": [
-             "Tests/Application/bin/console doctrine:database:create --if-not-exists",
-             "Tests/Application/bin/console doctrine:schema:update --force"
-         ],
-         "phpunit": [
-             "vendor/bin/phpunit"
-         ]
+        "bootstrap-test-environment": [
+            "Tests/Application/bin/adminconsole doctrine:database:create --if-not-exists --env test",
+            "Tests/Application/bin/adminconsole doctrine:schema:update --force --env test"
+        ],
+        "lint": [
+            "@phpstan",
```

Comment:
lint-twig is missing for the twig files in https://github.com/sulu/SuluCommentBundle/tree/2.x/Resources/views/WebsiteComment